copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+
+Request body:
+```js
+{
+  "data": {{$last}},
+  "action": "update",
+  "pass": "{{$env.GCALENDARHOOKSECRET}}"
+}
+
+```
+
+> note that {{$last}} is not quoted!
+
+***
+
+- Node 5 - "Condition"
+  
+![node 05](/copy-this-template-directory/directus_flow_3_05.png "node 05")
+
+Rules:
+
+```js
+{
+    "$last": {
+        "data": {
+            "res": {
+                "_eq": "need_update_calendar_event_id"
+            }
+        }
+    }
+}
+```
+
+
+
+***
+
+- Node 6 - "Update Data"
+
+![node 06](/copy-this-template-directory/directus_flow_3_06.png "node 06")
+
+Collection - a collection of your choice
+
+IDs (edit raw value):
+```js
+[
+    "{{$trigger.keys[0]}}"
+]
+```
+
+Payload:
+```js
+{
+    "calendar_event_id": "{{request_webhook_update.data.id}}"
+}
+```
+
+> Make sure that you are using the same key (here it's "request_webhook_update") as you set in node 4
+
+Query is empty
+
+***
+
+- Node 7 - "Condition"
+
+![node 07](/copy-this-template-directory/directus_flow_3_07.png "node 07")
+
+Rules:
+
+```js
+{
+    "$trigger": {
+        "event": {
+            "_ends_with": ".items.create"
+        }
+    }
+}
+```
+
+
+***
+
+- Node 8 - "Run Script"
+
+![node 08](/copy-this-template-directory/directus_flow_3_08.png "node 08")
+
+Key - **"payload_transformed"**
+
+Code
+```js
+module.exports = async function(data) {
+	let out = data.$trigger.payload;
+	out["id"] = data.$trigger.key;
+	return out;
+}
+```
+
+
+***
+
+- Node 9 - "Webhook / Request URL"
+
+![node 09](/copy-this-template-directory/directus_flow_3_09.png "node 09")
+
+key - **request_webhook_create**
+
+URL is `{{$env.GCALENDARHOOKURL}}` - the actual value in the environment variable will be set after Google Apps Script is published.
+
+Method is Post
+
+Request body:
+```js
+{
+  "data": {{$last}},
+  "action": "create",
+  "pass": "{{$env.GCALENDARHOOKSECRET}}"
+}
+```
+
+> note that {{$last}} is not quoted!
+  
+
+***
+
+- Node 10 - "Condition"
+  
+![node 10](/copy-this-template-directory/directus_flow_3_10.png "node 10")
+
+Rules:
+
+```js
+{
+    "$last": {
+        "data": {
+            "res": {
+                "_eq": "need_update_calendar_event_id"
+            }
+        }
+    }
+}
+```
+
+
+
+***
+
+- Node 11 - "Update Data"
+
+![node 11](/copy-this-template-directory/directus_flow_3_11.png "node 11")
+
+Collection - a collection of your choice
+
+IDs (edit raw value):
+```js
+[
+    "{{$trigger.key}}"
+]
+```
+
+Payload:
+```js
+{
+    "calendar_event_id": "{{request_webhook_create.data.id}}"
+}
+```
+
+> Make sure that you are using the same key (here it's "request_webhook_create") as you set in node 9
+
+Query is empty
+
+***
+
+&nbsp; 
+
+### Flow "Process events from Google Calendar"
+
+This is a webhook, called from Google Apps script when an event is created / updated / deleted in Google Calendar.
+
+Collection1 item will be Created / Updated / Deleted from incoming hook parameters.
+
+This flow is a bit wide, so the screenshot is split in two.
```

Comment:
```suggestion
This final Flow is the entry point for the Google Apps Script to interact with Directus regardless of whether it is triggered based on a create, update, or delete operation. It will determine and execute the correct operations within your collection.
```